src/proof-of-concept.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Assumes tracks are strings formatted "Song,Artist"
```

Comment:
Excellent notes! I think this is fine. If you want something that is a little easier to work with, I might recommend using a Map for your song data. That way, it's flexible in terms of what clients can store:

```json
{
    "title": "...",
    "artist": "...",
    "album": "...",
    "pathToSongFile": "..."
}
```

You can also make an inner Song class:

```java
private class Song (
  String title;
  String artist;
}
```